diagnostics/top_heaviness_metric/doc/top_heaviness_metric.rst

```diff
@@ -0,0 +1,90 @@
+Top-Heaviness Metric Diagnostic Documentation
+================================
+
+Last update: 5/30/2021
+
+The vertical profiles of diabatic heating have important implications for large-scale dynamics, especially for the coupling between the large-scale atmospheric circulation and precipitation processes. We adopt an objective approach to examine the top-heaviness of vertical motion (Back et al. 2017), which is closely related to the heating profiles and a commonly available model output variable. The diagnostic metric can also be used to evaluate the diabatic heating profile.
+
+Version & Contact info
+----------------------
+
+.. '-' starts items in a bulleted list:
+   https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists
+
+- Version/revision information: version 1.0 (5/21/2021)
+- Developer/point of contact (Jiacheng Ye, jye18@illinois.edu, DAS UIUC; Zhuo Wang, zhuowang@illinois.edu, DAS UIUC)
+
+.. Underline with '^'s to make a third-level heading.
+
+Open source copyright agreement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The MDTF framework is distributed under the LGPLv3 license (see LICENSE.txt).
+
+Functionality
+-------------
+
+The currently package consists of following functionalities:
+
+(1) Calculation of the fractional variance of vertical velocity at each grid point explained by two base functions (i.e., idealized deep convection profile, idealized deep stratiform profile)
+
+(2) Calculation of the top-heaviness ratio (O2/O1)
+
+(3) Other analysis (to be added soon)
```

Comment:
Can you be a bit more specific?   Are these "other analyses" planned for this current phase of the MDTF?

Reply:
After having a conversation with Zhuo Wang, we agree to just have step 1) and 2).